rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -935,6 +958,20 @@ private TrieStore buildMultiTrieStore(Path databasePath, String namePrefix, int
         );
     }
 
+    @VisibleForTesting
+    public static void mergeMultiTrieStoreDBs(Path trieStorePath, List<Path> multiTrieStorePaths) {
```

Comment:
same here

Reply:
I rather keep it this way. Maybe a different component should take care of of the whole `TrieStore` lifecycle, but doing that goes well beyond the scope of this PR. So far, being this method _static_, it should be pretty harmless and easy to refactor in the future.